### PR TITLE
hwdb: correct touchpad resolution for Asus N550JV

### PR DESCRIPTION
**Pull request by EliaGeretto**
Asus N550JV is affected by the exact same issue as X550CC and S550CB given that they use the same touchpad. For this reason, I updated the rule to match also this model. I tested the matching and the result on my own laptop.

### CODE REVIEW

File: hwdb/60-evdev.hwdb [hunk 1]
```diff
@@ -114,8 +114,8 @@ evdev:name:ETPS/2 Elantech Touchpad:dmi:bvn*:bvr*:bd*:svnASUSTeKComputerInc.:pnK
  EVDEV_ABS_35=::18
  EVDEV_ABS_36=::16
 
-# Asus X550CC and S550CB
-evdev:name:ETPS/2 Elantech Touchpad:dmi:*:svnASUSTeKCOMPUTERINC.:pn?550C?:*
+# Asus X550CC, S550CB and N550JV
+evdev:name:ETPS/2 Elantech Touchpad:dmi:*:svnASUSTeKCOMPUTERINC.:pn?550??:*
```
Comment (keszybz) [inline hwdb/60-evdev.hwdb]: I think it'd be safer to add a separate line. Assuming that all ASUSes with 550 in the name are the same might break in the future.

@whot?

Comment (EliaGeretto) [reply hwdb/60-evdev.hwdb]: Ok, I can make a separate rule. Given that I am at it, do you want me to break also X550CC and S550CB in two separate rules? I assume the same principle applies.

Comment (keszybz) [reply hwdb/60-evdev.hwdb]: That'd be great.

**Comment by EliaGeretto** [comment]
I force pushed two commits, one to split the rules and another to add a new one for N550JV. Let me know if everything is in order.

File: hwdb/60-evdev.hwdb [hunk 2]
```diff
@@ -114,8 +114,15 @@ evdev:name:ETPS/2 Elantech Touchpad:dmi:bvn*:bvr*:bd*:svnASUSTeKComputerInc.:pnK
...
 
-# Asus X550CC and S550CB
-evdev:name:ETPS/2 Elantech Touchpad:dmi:*:svnASUSTeKCOMPUTERINC.:pn?550C?:*
+# Asus X550CC
+evdev:name:ETPS/2 Elantech Touchpad:dmi:*:svnASUSTeKCOMPUTERINC.:pnX550CC:*
+ EVDEV_ABS_00=::31
+ EVDEV_ABS_01=::30
+ EVDEV_ABS_35=::31
+ EVDEV_ABS_36=::30
+
```
Comment (whot) [inline hwdb/60-evdev.hwdb]: you can actually have multiple match lines before the data. So this is valid but better in this case:
```
evdev:name:ETPS/2 Elantech Touchpad:dmi:*:svnASUSTeKCOMPUTERINC.:pnX550CC:*
evdev:name:ETPS/2 Elantech Touchpad:dmi:*:svnASUSTeKCOMPUTERINC.:pnS550CB:*
evdev:name:ETPS/2 Elantech Touchpad:dmi:*:svnASUSTeKCOMPUTERINC.:pnN550JV:*
  EVDEV_ABS=...
```

**Comment by EliaGeretto** [comment]
I force pushed two new commits, doing exactly the same thing, but in a single rule with multiple match lines, as requested by @whot.

**Pull request merged**